app/scripts/models/file-model.js

```diff
@@ -15,6 +15,7 @@ const FileModel = Backbone.Model.extend({
         name: '',
         keyFileName: '',
         passwordLength: 0,
+        confirmPasswordLength: 0,
```

Comment:
Please don't change the model for this feature, it must live in the UI and change the model when both passwords match.